packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -69,15 +71,62 @@ const confirmDialogsPropTypes = {
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+  selectedEntries,
+  slug,
+}) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+
+  const fetchDraftRelations = async () => {
+    try {
+      const endPoint = `/content-manager/collection-types/${slug}/actions/countManyEntriesDraftRelations?${stringify(
+        { ids: selectedEntries },
+        { encode: false }
+      )}`;
+
+      const draftRelations = await get(endPoint);
+      const draftEntries = await draftRelations.data.data;
+
+      return draftEntries;
+    } catch (err) {
+      console.error(err.message);
+
+      return null;
+    }
+  };
+
+  const { data } = useQuery(['draft-relations'], fetchDraftRelations, {
+    enabled: isOpen,
+  });
 
   return (
     <ConfirmBulkActionDialog
       isOpen={isOpen}
       onToggleDialog={onToggleDialog}
       dialogBody={
         <>
+          {data && (
+            <Typography id="confirm-description">
```

Comment:
Do you need the id?

Reply:
it is used here https://github.com/strapi/strapi/blob/4522fcab25fb1dce1591acc477b52322cdd9b14f/packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js#L31